### PR TITLE
Logging (--verbose), refactor of protocol work 

### PR DESCRIPTION
**Pull request by paulirish**

### CODE REVIEW

**Comment by paulirish** [comment]
Still a few things to clean up, i'll put up another commit in an hour.

**Comment by paulirish** [comment]
ready for review.

File: audits/mobile-friendly/viewport.js [hunk 1]
```diff
@@ -26,6 +26,8 @@ class Viewport {
   }
 
   static audit(inputs) {
+    // FIXME: This'll be a false positive if the body content has this string.
```
Comment (samccone) [inline audits/mobile-friendly/viewport.js]: TODO:

change to: TODO

File: helpers/browser/driver.js [hunk 2]
```diff
@@ -59,8 +60,9 @@ class ChromeProtocol {
         return resolve(this._chrome);
       }
 
-      chromeRemoteInterface({}, chrome => {
+      chromeRemoteInterface({port: 9222}, chrome => {
```
Comment (samccone) [inline helpers/browser/driver.js]: let's default look to the env and default to 9222

Comment (samccone) [inline helpers/browser/driver.js]: process.env['.....'] || ....

**Comment by paulirish** [comment]
comments addressed

**Comment by brendankenny** [comment]
looks great to me

**Pull request merged**